### DISCUSSION TITLE
How does this work in VS Code ?

### DISCUSSION THREAD

**Discussion by jassMarok** [mkosir/express-typescript-typeorm-boilerplate#114] [Q&A] [answered]
Typescript is showing errors for missing packages, I am running the boilerplate with docker and the node_modules folder is empty as those are installed on the docker instance separately. I looked into creating the [devcontainer.json ](https://code.visualstudio.com/docs/remote/containers#_quick-start-open-an-existing-folder-in-a-container) but not sure if that is the recommended way to go, also by doing this I lose all the git functionality in VS Code.

**Comment by mkosir** [comment, marked as answer]
Hey @jassMarok, for local development so IDE can recognize the packages and infer types for intellisense support you must first do `npm install`.

**Reply by jassMarok** [reply]
I was also able to run this with the Remote Containers Extension for VS Code, had to install the git on the container itself to get the git runner but other than that everything is running smoothly. I can create a PR if that's something you would be interested in.
